### PR TITLE
docs: updated generated-index cards

### PR DESCRIPTION
**Pull request by huynhicode**
# Pull Request

## 📖 Description

This PR:
- removes the default document and directory icons from the generated-index cards
- updates the descriptions and styling of the generated-index cards

The generated-index card descriptions were previously auto generated from the content at the top of the respective markdown file.  The descriptions have now been updated either by adding description front matter, or by adding a custom prop description to the sidebars.js file.  

To access the DocCard (in order to update the generated-index cards), the component was swizzled and ejected.
- swizzling: allows for a theme component to be swapped with custom implementation
- ejecting: creates a copy of the original component, which can then be customized
- reference: https://docusaurus.io/docs/swizzling

Styling:
- card container is now larger
- card on hover effect is now the primary color #fb356d
- no icons

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## ⏭ Next Steps

---

## Screenshots:

**desktop**:

Screen Shot 2022-04-08 at 6 43 56 PM

Screen Shot 2022-04-08 at 6 29 51 PM

Screen Shot 2022-04-08 at 6 31 29 PM

**mobile**: 

Screen Shot 2022-04-08 at 6 44 25 PM

Screen Shot 2022-04-08 at 6 30 00 PM

### CODE REVIEW

File: packages/web-components/fast-foundation/src/number-field/README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ id: number-field
 title: fast-number-field
 sidebar_label: number-field
 custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-foundation/src/number-field/README.md
+description: fast-number-field is a web component implementation of a text field.
```
Comment (EisenbergEffect) [inline packages/web-components/fast-foundation/src/number-field/README.md:6]: I think this should say "number input" or something like that at the end.

File: sites/website/src/docs/resources/why-web-components.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ id: why-web-components
 title: Why Web Components?
 sidebar_label: Why Web Components?
 custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/resources/why-web-components.md
+description: Performance, interoperability, compatibility are a few reasons you might choose Web Components.
```
Comment (EisenbergEffect) [inline sites/website/src/docs/resources/why-web-components.md:6]:
```suggestion
description: Performance, interoperability, and compatibility are a few reasons you might choose Web Components.
```

File: packages/web-components/fast-element/docs/guide/observables-and-state.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ id: observables-and-state
 title: Observables and State
 sidebar_label: Observables and State
 custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-element/docs/guide/observables-and-state.md
+description: Now that you're familiar with the robust and powerful features of FASTElement, you're ready to build your own components and apps.
```
[the remark this answers is not in the document]
Comment (huynhicode) [reply packages/web-components/fast-element/docs/guide/observables-and-state.md]: Thank you @EisenbergEffect. 

Description updated to:

"fast-number-field is a web component implementation of a text field number input."

File: packages/web-components/fast-foundation/src/number-field/README.md (hunk 1 above)
Comment (huynhicode) [reply packages/web-components/fast-foundation/src/number-field/README.md]: Thank you @EisenbergEffect. 

Description updated to:

"fast-number-field is a web component implementation of a text field number input."

**Comment by bheston** [comment]
> > Found a couple issues in the descriptions. Also, can you check with Brian on the styles he wants for the hovered cards? Thanks Steph!
> 
> Thank you @EisenbergEffect! Yes, will reach out to @bheston for confirmation re: on hover effect.

Yes, I think increasing the shadow effect on hover is an acceptable treatment here.

**Review by EisenbergEffect — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
@huynhicode If you can resolve the conflict I think we can go ahead and merge this. Thanks!

**Comment by huynhicode** [comment]
Closing this PR as I was unable to resolve the yarn.lock merge conflicts.  Opened a new one PR here: https://github.com/microsoft/fast/pull/5898.

**Pull request closed without merging**